include/deal.II/numerics/vector_tools.h

```diff
@@ -2156,24 +2156,57 @@ namespace VectorTools
 
   /**
    * Create a right hand side vector for a point source at point @p p. In
-   * other words, it creates a vector $F$ so that $F_i = \int_\Omega
+   * other words, it creates a vector $\mathbf F$ so that $F_i = \int_\Omega
```

Comment:
I am not sure if we are using bold symbols for vectors representing values related to degrees of freedom consistently. At least, `step-3` doesn't.
In any case, the formatting used here should be consistent with the documentation of `VectorTools::project` (line 804), the one for the other `VectorTools::create_point_vector` variant (line 2255) and the one for `VectorTools::integrate_difference` (line 2393).